packages/mobile/src/screens/coin-details-screen/components/BalanceCard.tsx

```diff
@@ -69,33 +69,33 @@ const HasBalanceState = ({
   onBuy,
   onSend,
   onReceive,
-  mint
-}: BalanceStateProps & { mint: string }) => {
+  mint,
+  coinName
+}: BalanceStateProps & { mint: string; coinName: string }) => {
   const { tokenBalanceFormatted, formattedHeldValue } =
     useFormattedTokenBalance(mint)
 
   return (
     <Flex column gap='l' w='100%'>
-      <Flex row gap='s' alignItems='center'>
-        <TokenIcon logoURI={logoURI} size={64} />
-        <Flex column gap='xs'>
-          <Flex row gap='xs'>
-            <Text variant='heading' size='l' color='default'>
-              {tokenBalanceFormatted}
-            </Text>
-            <Text variant='heading' size='l' color='subdued'>
-              {title}
+      <Flex row alignItems='center' justifyContent='space-between'>
+        <Flex row alignItems='center' gap='l' style={{ flexShrink: 1 }}>
+          <TokenIcon logoURI={logoURI} size={64} />
+          <Flex column gap='2xs'>
+            <Text variant='heading' size='s'>
+              {coinName}
             </Text>
+            <Flex row gap='xs' alignItems='center'>
+              <Text variant='title' size='l'>
+                {tokenBalanceFormatted}
+              </Text>
+              <Text variant='title' size='l' color='subdued'>
+                {title}
+              </Text>
+            </Flex>
           </Flex>
-          <Text
-            variant='heading'
-            size='s'
-            color='subdued'
-            style={{
-              lineHeight: spacing.unit7,
-              transform: [{ translateY: -spacing.unitHalf }]
-            }}
-          >
+        </Flex>
+        <Flex row alignItems='center' gap='m' ml={spacing.unit22}>
+          <Text variant='title' size='l' color='default'>
```

Comment:
intentionally undoing the lineHeight + translate to save the subscript from being clipped?